manifests/example/vddk-configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: v2v-vmware
+data:
+  vddk-init-image: registry:5000/vddk-init:latest
```

Comment:
No, there can't be a public location for this container because it holds the non-redistributable VDDK libraries. The intent is that the user will have to create it and upload it to their own registry. I put some extra comments in here to clarify this as suggested.